From 6e25ac99d08adf519252c48932b16479e080b2c3 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Wed, 31 Oct 2018 17:07:40 +0800
Subject: [PATCH 237/741] sdk_fman: disable ptp timer probe

commit b4eb63644e862e0b84fc7509fd5b20da4603b8f2 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

DPAA PTP timer was managed by ptp_qoriq driver in drivers/ptp/.
We will no longer manage it in sdk_fman driver and use related
APIs.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
index fb5306bf0780..9560ac071475 100755
--- a/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/wrapper/lnxwrp_fm.c
@@ -682,6 +682,11 @@ static t_LnxWrpFmDev * ReadFmDevTreeNode (struct platform_device *of_dev)
         }
     }
 
+/* DPAA PTP timer was managed by ptp_qoriq driver in drivers/ptp/.
+ * We will no longer manage it in sdk_fman driver and use related
+ * APIs.
+ */
+#if 0
     /* Get the RTC base address and size */
     memset(ids, 0, sizeof(ids));
     if (WARN_ON(strlen("ptp-timer") >= sizeof(ids[0].name)))
@@ -704,6 +709,7 @@ static t_LnxWrpFmDev * ReadFmDevTreeNode (struct platform_device *of_dev)
             p_LnxWrpFmDev->fmRtcMemSize = res.end + 1 - res.start;
         }
     }
+#endif
 
 #if (DPAA_VERSION >= 11)
     /* Get the VSP base address */
-- 
2.17.1

